src/main/resources/liquibase/crop_changelog/14_1_0.xml

```diff
@@ -0,0 +1,239 @@
+    </changeSet>
+    <changeSet author="gelli" id="v14.1.0-16">
+        <preConditions onFail="MARK_RAN">
+            <columnExists tableName="methods" columnName="muid_bkp"/>
+        </preConditions>
+        <comment>Drop Column muid_bkp if exists from table methods.
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+            ALTER TABLE methods DROP COLUMN muid_bkp;
+        </sql>
+    </changeSet>
+    <changeSet author="gelli" id="v14.1.0-17">
+        <preConditions onFail="MARK_RAN">
+            <columnExists tableName="names" columnName="nuid_bkp"/>
+        </preConditions>
+        <comment>Drop Column nuid_bkp if exists from table names.
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+            ALTER TABLE names DROP COLUMN nuid_bkp;
+        </sql>
+    </changeSet>
+    <changeSet author="gelli" id="v14.1.0-18">
+        <preConditions onFail="MARK_RAN">
+            <columnExists tableName="udflds" columnName="fuid_bkp"/>
+        </preConditions>
+        <comment>Drop Column fuid_bkp if exists from table udflds.
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+            ALTER TABLE udflds DROP COLUMN fuid_bkp;
+        </sql>
+    </changeSet>
+    <changeSet author="gelli" id="v14.1.0-19">
+        <preConditions onFail="MARK_RAN">
+            <columnExists tableName="reflinks" columnName="refuid_bkp"/>
+        </preConditions>
+        <comment>Drop Column refuid_bkp if exists from table reflinks.
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+            ALTER TABLE reflinks DROP COLUMN refuid_bkp;
+        </sql>
+    </changeSet>
+    <changeSet author="gelli" id="v14.1.0-20">
+        <preConditions onFail="MARK_RAN">
+            <columnExists tableName="projectprop" columnName="projectprop_users_bkp"/>
+        </preConditions>
+        <comment>Drop Column projectprop_users_bkp if exists from table projectprop.
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+            ALTER TABLE projectprop DROP COLUMN projectprop_users_bkp;
+        </sql>
+    </changeSet>
+    <changeSet author="gelli" id="v14.1.0-21">
+        <preConditions onFail="MARK_RAN">
+            <columnExists tableName="nd_geolocationprop" columnName="nd_geolocationprop_users_bkp"/>
+        </preConditions>
+        <comment>Drop Column nd_geolocationprop_users_bkp if exists from table nd_geolocationprop.
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+            ALTER TABLE nd_geolocationprop DROP COLUMN nd_geolocationprop_users_bkp;
+        </sql>
+    </changeSet>
+    <changeSet author="gelli" id="v14.1.0-22">
+        <preConditions onFail="MARK_RAN">
+            <tableExists tableName="users_bkp"/>
+        </preConditions>
+        <comment>Drop Table users_bkp if exists.
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+            DROP TABLE IF EXISTS users_bkp;
```

Comment:
you don't need the IF EXISTS in DROP TABLE clauses since changeset will only run if precondition criteria was met

Reply:
Noted. Will remove the If Exists in sql syntax